src/soter/openssl/soter_ec_key_3.c

```diff
@@ -202,6 +194,11 @@ soter_status_t soter_engine_specific_to_ec_priv_key(const soter_engine_specific_
     res = SOTER_SUCCESS;
 
 err:
+    if (res != SOTER_SUCCESS) {
+        soter_wipe(bignum_buf, sizeof(bignum_buf));
+    }
```

Comment:
It seems like we don't use this buffer after the `err` label, so we can wipe it in any case, not only if there is an error.

Reply:
Wow, good point. Removed the `if`, here and in RSA priv key serializatoin